Taskfile.yml

```diff
@@ -97,9 +98,9 @@ tasks:
     aliases: ["build"]
     vars:
       BINARY: terraform-provider-opslevel_{{.VERSION_DYNAMIC}}
-      LOCATION: "${HOME}/.terraform.d/plugins/registry.terraform.io/opslevel/opslevel/{{.VERSION_DYNAMIC}}/{{OS}}_{{ARCH}}"
+      LOCATION: "{{.PLUGIN_DIR}}/{{.VERSION_DYNAMIC}}/{{OS}}_{{ARCH}}"
     cmds:
-      - go build -ldflags="-s -w -X github.com/opslevel/terraform-provider-opslevel/opslevel.version={{.VERSION_DYNAMIC}}" -o {{.BINARY}} || exit 1
+      - go build -ldflags="-s -w -X main.version={{.VERSION_DYNAMIC}}" -o {{.BINARY}} || exit 1
```

Comment:
This is giving me flashbacks to the release process.  Which way is the right way again?! 😅

Reply:
I think this is right because its in the `main.go` now and its pushed into the provider during `func main()` but that also means that this file can be deleted as its no longer used. https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/opslevel/version.go

Reply:
Lastly - if we override this to be the `VERSION_DYNAMIC` that means our API will get blown up with lots of different user-agent versions when we test locally.  This translates into tags in datadog making use pay more for customer metrics.  The reason it defaults to `dev` is so that all local development comes from that version instead.  So i think this override is unnecessary and we can remove it.

Reply:
It's `main.version` 😁 we added it in the [goreleaser config](https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/.goreleaser.yml#L12) but it wasn't updated here

Reply:
Loud and clear I'll drop `VERSION_DYNAMIC`

Reply:
Actually we need `VERSION_DYNAMIC`, so I'm updating it to use the latest git tag and appending `5` to it